geoportailv3/static/js/interrogation/interrogationdirective.js

```diff
@@ -160,21 +160,26 @@ app.InterrogationController = function($scope, $http,
     }
   },this));
 
-  /*TODO : should implement CORS header before using it*/
-  /*
+
   map.on('pointermove', goog.bind(function(evt) {
     if (evt.dragging) {
       return;
     }
-
-
     var pixel = map.getEventPixel(evt.originalEvent);
     var hit = map.forEachLayerAtPixel(pixel, function(layer) {
-      return true;
+      if (goog.isDefAndNotNull(layer)) {
```

Comment:
Yes, it could happens when an overlay layer is existing. I'm not sure if it's normal or not but it happens.